### PR TITLE
Move data schemas to basic concepts section

### LINKED ISSUE #1196: Move Data Schemas to Basic Concepts section
[Data Schemas](https://w3c.github.io/vc-data-model/#data-schemas) is currently in the [advanced concepts](https://w3c.github.io/vc-data-model/#advanced-concepts) section.

I believe it is not more advanced than securing a credential, or representing the status of a credential both of which are in the [basic concepts](https://w3c.github.io/vc-data-model/#basic-concepts) section of the data model.

Given the adopted work item [vc-json-schema](https://github.com/w3c/vc-json-schema) I would like to propose moving the section to Basic Concepts to continue promote its usage and utility in the VC space. Notably, data schemas are already used in credential deployments such as [the EBSI project](https://ec.europa.eu/digital-building-blocks/code/projects/EBSI/repos/json-schema/browse), and upcoming deployments by TBD and others.

### PR DESCRIPTION
**Pull request by decentralgabe**
Fix #1196 
No changes besides the move.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1207.html" title="Last updated on Jul 25, 2023, 5:51 PM UTC (8321efd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1207/d84cf2e...8321efd.html" title="Last updated on Jul 25, 2023, 5:51 PM UTC (8321efd)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2086,6 +2086,159 @@ <h4>Presentations Using Derived Credentials</h4>
...
+    "alumniOf": {
+      "name": "Example University"
+    }
+  },
+  <span class="highlight">"credentialSchema": [{
+    "id": "https://example.org/examples/degree.json",
+    "type": "JsonSchema2023"
+  },
+  {
+    "id": "https://example.org/examples/person.json",
```
Comment (msporny) [inline index.html:2181]:
```suggestion
    "id": "https://example.org/examples/alumni.json",
```

I know this isn't a part of this PR, but might want to fix it just for understandability.

**Review by msporny — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -2086,6 +2086,159 @@ <h4>Presentations Using Derived Credentials</h4>
           </figure>
         </section>
       </section>
+
+<section>
```
Comment (davidlehn) [inline index.html:2090]: Indent to proper level.

**Comment by davidlehn** [comment]
The git history here got messed up.  Looks like the most recent merge commit added back the original section.

File: index.html (hunk 2 above)
Comment (decentralgabe) [reply index.html]: done

**Comment by decentralgabe** [comment]
@davidlehn done

**Review by iherman — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Comment by decentralgabe** [comment]
Open a week. Changes addressed. Multiple approvals, no remaining conflicts. Merging.

**Pull request merged**